SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialLoadBalancingHandler.h

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "EngineClasses/SpatialNetDriver.h"
+#include "Utils/SpatialActorUtils.h"
+
```

Comment:
Might be worth having an exemplar context that matches the context concept with comments.